src/pdfform.cpp

```diff
@@ -655,3 +655,8 @@ wxPdfDocument::LoadZapfDingBats()
     m_fontSize    = saveSize / m_k;
   }
 }
+
+double wxPdfDocument::GetHeaderHeight() const
```

Comment:
The implementation of `wxPdfDocument::GetHeaderHeight()` has nothing to do with PDF forms. Therefore it should be moved to `pdfdocument.cpp`. (Since the method is protected, another possibility would be to move it to `pdfkernel.cpp`, but I think `pdfdocument.cpp` is more suitable, as we may consider to make this method public at a later time.)

Reply:
Yes, you're absolutely correct. Somehow the C++ Resharper played me a prank. I declared it in ```pdfdocument.h``` and just clicked on "create definition" and didn't reviewed the source file. I assumed it is in ```pdfdocument.cpp```. I'll adjust it, but probably not today.

Reply:
Thanks in advance. (It's not urgent ...)